website/docs/cloud-docs/projects/managing.mdx

```diff
@@ -55,15 +55,16 @@ To edit a project:
 1. Click on a project name of the project you want to edit.
 1. Click **Settings**.
 
-On this **General settings** page, you can update the project name, project
-description, and delete the project. On the **Team access** page, you can modify
-team access to the project.
+On this **General settings** page, you can update the project name and project
+description, override the default project, and delete the project. On the **Team
+access** page, you can modify team access to the project.
 
 ## Delete a Project
 
-You can only delete projects that do not contain workspaces.
+You can only delete a project that does not contain workspaces and is not the
+default project.
```

Comment:
In the future, we will allow deleting a default project but that isn't the case today.